roles/pulp_common/scripts/move-artifacts-to-media-root.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# This script moves a directory passed in the first parameter to the location passed in as a second
+# parameter. It then creates a symlink from the old location to the new. When SELinux is enabled,
+# SELinux contexts are preserved.
+
+# Fail if the mv or ln fail
+set -e
+
+mv -Z $1 $2
+ln -s $2 $1
+
+# Don't fail if 'getenforce' command is not present
+set +e
+
+command -v getenforce
+
+if [ $? == 0  ]; then
+  set -e
+  if [ `getenforce` == 'Permissive' ] || [ `getenforce` == 'Enforcing' ]; then
+    echo foo
```

Comment:
https://www.thegeekdiary.com/how-to-check-whether-selinux-is-enabled-or-disabled/ suggests using `selinuxenabled` and checking the exit code. You can use `which` to check if the command is available.

Reply:
Yeah, use selinuxenabled.

But per a previous comment which he already addressed, he is using `command -v` rather than `which`. Because it is part of the bash package rather than a separate `which` package that is not installed in minimal images.